scratch/logging-ns3-script.cc

```diff
@@ -0,0 +1,23 @@
+#include <iostream>
+#include <fstream>
+#include <string>
+
+#include "ns3/core-module.h"
+
+using namespace ns3;
+
+NS_LOG_COMPONENT_DEFINE ("Logger");
+
+int
+main (int argc, char *argv[])
+{
+
+  Simulator::Run ();
+  NS_LOG_DEBUG("Debug");
+  NS_LOG_INFO("Info");
+  NS_LOG_WARN("Warn");
+  NS_LOG_ERROR("Error");
+  NS_LOG_LOGIC("Logic");
+  NS_LOG_FUNCTION("Function");
+  Simulator::Destroy ();
```

Comment:
make sure to follow ns-3 code style guidelines. you can invoke the utils/check-style.py script to do so.
I would suggest something like:
```
python utils/check-style.py -i -l 3 -f scratch/logging-ns3-script.cc
```
NOTE: i haven't tested this, just remembering more or less the syntax